src/components/show-edit-pod/show-edit-pod.test.js

```diff
@@ -0,0 +1,323 @@
+      (boolVal, backgroundColor) => {
+        CypressMountWithProviders(
+          <ShowEditPodComponent softDelete={boolVal} />
+        );
+        showEditPod().should("have.css", "background-color", backgroundColor);
+      }
+    );
+
+    it.each([
+      [false, "be.visible"],
+      [true, "not.exist"],
+    ])(
+      "should check when hideDeleteButtonInViewMode is %s for ShowEditPod component",
+      (boolVal, state) => {
+        CypressMountWithProviders(
+          <ShowEditPodComponent hideDeleteButtonInViewMode={boolVal} />
+        );
+        HideDeleteButtonShowEditPod().should(state);
+      }
+    );
+
+    it("should check editFields for textbox component", () => {
+      CypressMountWithProviders(
+        <ShowEditPodComponent editing editFields={<Textbox label="Field" />} />
+      );
+      editFieldsShowEditPod().should("be.visible");
+    });
+
+    it.each([
+      ["right", "end"],
+      ["left", "start"],
+    ])(
+      "should check when buttonAlign is %s for ShowEditPod component",
+      (align, webkitBoxPack) => {
+        CypressMountWithProviders(
+          <ShowEditPodComponent editing buttonAlign={align} />
+        );
+        FormFooterShowEditPod().should(
+          "have.css",
+          "-webkit-box-pack",
+          webkitBoxPack
+        );
+      }
+    );
+
+    it.each([
+      [true, "be.visible"],
+      [false, "not.exist"],
+    ])(
+      "should check when cancel button is %s for ShowEditPod component",
+      (boolVal, state) => {
+        CypressMountWithProviders(
+          <ShowEditPodComponent editing cancel={boolVal} />
+        );
+
+        CancelButtonShowEditPod().should(state);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check cancelText as %s for ShowEditPod component",
+      (cancelText) => {
+        CypressMountWithProviders(
+          <ShowEditPodComponent editing cancelText={cancelText} />
+        );
+
+        CancelButtonShowEditPod().should("have.text", cancelText);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check saveText as %s for ShowEditPod component",
+      (saveText) => {
+        CypressMountWithProviders(
+          <ShowEditPodComponent editing saveText={saveText} />
+        );
+        SaveButtonShowEditPod().should("have.text", saveText);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check deleteText as %s for ShowEditPod component",
+      (deleteText) => {
+        CypressMountWithProviders(
+          <ShowEditPodComponent editing deleteText={deleteText} />
+        );
+        DeleteButtonShowEditPod().should("have.text", deleteText);
+      }
+    );
+
+    it.each([
+      [
+        false,
+        "rgb(0, 126, 69) none repeat scroll 0% 0% / auto padding-box border-box",
+      ],
+      [
+        true,
+        "rgb(230, 235, 237) none repeat scroll 0% 0% / auto padding-box border-box",
+      ],
+    ])(
+      "should check saveText as %s for ShowEditPod component",
```

Comment:
I think the title of this test is misleading. It should be changed to something like "should check saving state is %s for ShowEditPod component".